test/api.js

```diff
@@ -2286,13 +2286,13 @@ describe('/v1', function() {
         return getUniqueUserAndToken(client1Id.toString('hex'), {
           uid: user1.uid,
           email: user1.email,
-          scopes: ['profile']
+          scopes: ['profile', 'profile:write']
```

Comment:
@divyabiyani could we please add a new test just for this with more advanced cases, such as:

Same client:

1) scopes: ['profile', 'profile:write']
2) scopes: ['clients:write']
3) scopes: ['basket:write', 'profile:email']
4) scopes: ['profile:uid', 'profile', 'profile:write']

In this new test we just care about scopes, so we don't need the extra assertions.